librocksdb_sys/crocksdb/c.cc

```diff
@@ -4680,4 +4685,36 @@ uint64_t crocksdb_perf_context_env_new_logger_nanos(crocksdb_perf_context_t* ctx
   return ctx->rep.env_new_logger_nanos;
 }
 
+void crocksdb_run_ldb_tool(int argc, char** argv, char** errptr) {
+  errptr = nullptr;
```

Comment:
So why pass the `errptr` in?

Reply:
Just keep the code style equal with the way other functions use `ffi_try!`.

Reply:
I think it is not necessary, why do we need to keep it consistent to use `ffi_try!`?

Reply:
I've fixed it.